src/Data/List/Membership/Propositional/Properties.agda

```diff
@@ -331,16 +331,16 @@ finite inj (x ∷ xs) fᵢ∈x∷xs = excluded-middle helper
     lemma i≤j i≰1+j refl = i≰1+j (≤-step i≤j)
 
     f′ⱼ∈xs : ∀ j → f′ j ∈ xs
-    f′ⱼ∈xs j with i ≤? j
-    ... | yes i≤j = ∈-if-not-i (<⇒≢ (s≤s i≤j))
-    ... | no  i≰j = ∈-if-not-i (<⇒≢ (≰⇒> i≰j) ∘ sym)
+    f′ⱼ∈xs j with i ≤ᵇ j | ≤ᵇ⇒≤ i j | ≤⇒≤ᵇ {i} {j}
```

Comment:
This `| ≤ᵇ⇒≤ i j | ≤⇒≤ᵇ {i} {j}` is the pattern that
`Reflects.invert (≤ᵇ-reflects-≤ i j)` is abstracting over.